src/plugins/auto/auto_schedule.cpp

```diff
@@ -200,7 +202,8 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
                     if (contextPtr->workName.empty()) {
                         contextPtr->workName = contextPtr->deviceInfo.deviceName;
                     }
-                    GenerateWorkers(contextPtr->workName, contextPtr->executableNetwork);
+                    if (!isCumulative)
```

Comment:
@wgzintel  please help review this logic

Reply:
approved